openedx_webhooks/github_views.py

```diff
@@ -46,29 +49,34 @@ def pull_request():
         return "PONG"
 
     pr = event["pull_request"]
+    PR_NUMBER = pr['number']
```

Comment:
👍 re: *understanding-at-a-distance*

Thank you for taking the time to think this through, @nedbat. And `const` in JS does not equate to immutability, but is rather about reassignment of the variable. There's a debate in the JS community about the need to uppercase those names as well.